engine/classes/Elgg/Database/Plugins.php

```diff
@@ -410,11 +402,10 @@ function find($status = 'active') {
 			'type' => 'object',
 			'subtype' => 'plugin',
 			'limit' => ELGG_ENTITIES_NO_VALUE,
-			'selects' => array('plugin_oe.*', 'ps.value'),
```

Comment:
I mean `__get` and `__set` in `ElggPlugin` are reading private settings and not metadata. You need to add handling of `title` and `description` separately. IMO, not sure 100%.